fpga/mcs.wake

```diff
@@ -118,20 +118,18 @@ global def makeMCS plan =
 
     makeVivadoPlan sourceTcl outputDir
     | setVivadoPlanTclArgs tclArgs
-    | setVivadoPlanFnOutputs (\_ outputFile, Nil)
+    | setVivadoPlanFnOutputs (filter (matches `.*\.mcs`))
     | setVivadoPlanResources resources
     | setVivadoPlanVisibleFiles (outputDir.mkdir, bitstream, programImgList ++ tclSupportFiles)
     | runVivado
 
   vivadoMCS
   | getVivadoOutputAllOutputs
-  | head
-  | getOrElse "MCS failed: {outputFile}".makeError.makeBadPath
   | MCSOutputs plan
 
 tuple MCSOutputs =
   Plan_: MCSPlan
-  MCS_: Path
+  MCS_: List Path
```

Comment:
can we add another field instead of changing the type of an existing field to maintain backwards compatibility? like a new `AllMCS` field that is a list of the `*.mcs` files, and the existing `MCS` field still points to whatever file it was pointing at before